### PR TITLE
Feat: risk filter

### LINKED ISSUE #248: Create List of Strategies that are Missed In Risk Framework
We realized some strategies were not included in the Risk Framework (RF). 

They were not add to the withdrawal queue. So when we filtered the strategies by name using a contract, it didn't find the strategy.

The ouput of this task is:

Create a list of strategies that are not found in the RF.

Maybe we can get the full list of strategies from the SDK (check how we get the data at the [home page](https://yearn.watch/)). 
Then when the RF filters the groups, create a new list with the strategies that were not included.

The new list might be in a new page, adding a link the RF page depending on the network.

Note:

There are 126 strategies In Fantom [home page](https://yearn.watch/network/fantom).
But there are only 31 in the [RF page](https://yearn.watch/network/fantom/risk).

Some missing strategies were retired from the vaults (queue index -1), but we need to know those strategies that are still active but are not in the RF.

### PR DESCRIPTION
**Pull request by 0xkofee**
- The main page differs from the risk page because the risk page is using a smart contract to find strategy addresses while the front page uses a subgraph (previously an api). These two lists may not match. Although now that I fixed the "other" bug, all non categorized strategies now are lumped into that group.
- Filter works like the vaults with warnings filter. It filters the vault list if it contains a strategy that is not in the risk framework
- To find the strategy with the issue you would need to look for a new description on the strategy called `Missing Risk Profile` which is set to True or False. The strategies in the risk profile can be found from this smart contract: https://etherscan.io/address/0x2114d9a16da30fa5b59795e4f8c9ead19e40f0a0#readContract call `assetsStrategiesAddresses`
- ~~The contract call to get strategies in the "others" section of the risk framework takes forever/never resolves. I'm not sure why but even the risk page stopped resolving for me(edit: I found out that it wasn't resolving because the risk page also tried to load the front page vaults because of react). This others call is also what makes the risk page slow. I filtered it out for now. Maybe someone else has a better idea or we can leave it filtered out for now...~~

There are a few bugs that I found while working on this feature
- The "other" section on the risk profile doesn't show all strategies. It just shows the strategies from the first 25 vaults since the method `getAllStrategies` returns just the first page of results. After handling pagination, this number is much bigger. After fixing this bug, the front page risk filter no longer filters anything because the risk page has all strategies accounted for in the bigger "other" section. I went ahead and filtered out the others section when doing the comparison. I am assuming the "other" section just means we don't have a risk profile for them.
- Another thing I noticed was that the memoized functions were't really memoized as we were creating new memozied callbacks on demand. Will probably fix this in another PR.

### CODE REVIEW

**Review by 0xkofee — COMMENTED** [review]
I need to revert this before merge but for now I will leave it in for testing

File: src/utils/config/ethereum.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ export const mainnet: NetworkConfig = {
     toAddressExplorerUrl: (token: string): string =>
         `https://etherscan.io/address/${token}`,
     toTxExplorerUrl: (tx: string): string => `https://etherscan.io/tx/${tx}`,
-    subgraphUrl: `https://gateway.thegraph.com/api/${env.theGraphKey}/subgraphs/id/5xMSe3wTNLgFQqsAc5SCVVwT4MiRb5AogJCuSN9PjzXF`,
```
Comment (0xkofee) [inline src/utils/config/ethereum.ts]: I need to revert this before merge but I will leave it in for testing

File: src/utils/vaultMappings.ts [hunk 2]
```diff
@@ -79,6 +83,18 @@ export const mapVaultApiDataToVault = async (
         }
     );
 
+    const noOtherGroups = groups.filter((g) => g.id !== 'others'); // TODO why is others not resolving?
```
Comment (0xkofee) [inline src/utils/vaultMappings.ts]: I have no idea why the others group never resolves for me. The risk page makes the same call and it also never resolves the page for me...

File: src/utils/config/ethereum.ts (hunk 1 above)
Comment (magikadev) [reply src/utils/config/ethereum.ts]: Revert now please.

File: src/components/app/StrategiesList/index.tsx [hunk 3]
```diff
@@ -400,6 +400,22 @@ const _StrategiesList = (props: StrategiesListProps) => {
...
+                                                        <Grid
+                                                            item
+                                                            xs={12}
+                                                            md={2}
+                                                        >
+                                                            <StyledTitle>
+                                                                {vault &&
+                                                                strategy.isMissingRisk
+                                                                    ? 'true'
+                                                                    : 'false'}
```
Comment (magikadev) [inline src/components/app/StrategiesList/index.tsx]: 'True' / 'False' will be better than 'true' / 'false'

**Review by magikadev — COMMENTED** [review]
LGTM!

**Comment by storming0x** [comment]
I dont see any issue on the implementation, just would prefer for the output of this new data to be enclosed in a new page, not a fan of adding alerts data on the strategies that eventually will go away onces fixed, but it adds loading time and noise to existing strat list.

An alerts page may be a better place for this data, since its data that we can look at on demand and is not supposed to be the case that any strategy should be outside the risk framework, is just for outside case we missed smth in configuration.

**Comment by storming0x** [comment]
is this deprecated in favor of #288

**Comment by magikadev** [comment]
> is this deprecated in favor of #288

We are in favor of #288, so it's okay to close this PR I think.

**Pull request closed without merging**